.github/workflows/k8s-ci.yml

```diff
@@ -0,0 +1,22 @@
+name: K8s CI
+on:
+  pull_request:
+    types: ['opened', 'edited', 'reopened', 'synchronize']
```

Comment:
Can you comment this till we are ready with with install and basic io test ?

Reply:
Sure